### ISSUE TITLE
Allow logo in top navigation to be a link

### ISSUE THREAD

**Issue by rpmcoding** [Budibase/budibase#7935]
Standard practice for logos in navigation to be a link to the homepage.  Currently the logo cannot be made into a link.

**Comment by rpmcoding** [comment]
Bump

**Comment by matteoveglia** [comment]
Agreed! In my case I instead just made my own navigation and added a clickable container encompassing my logo and app name.

The only downside to this is that now I have to manage a nav pane and keep that manually 'synced' across screens
